### PR TITLE
[bitnami/redmine]: Allow specyfing domain

### PR DESCRIPTION
**Pull request by dolohow**
Sometimes domain name is different than effective user.  An exaple could be an AWS SES service, when you have username equal to AWS_ACCESS_KEY which does not even have domain part.

### Description of the change

This change allow specyfing env variable SMTP_DOMAIN, in the lack of this variable default, previous value will be used.

### Applicable issues

- https://github.com/bitnami/charts/issues/7241

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the contribution! Could you fix the DCO issue?

**Comment by dolohow** [comment]
> Thank you so much for the contribution! Could you fix the DCO issue?

Done

File: bitnami/redmine/5/debian-11/rootfs/opt/bitnami/scripts/libredmine.sh [hunk 1]
```diff
@@ -172,7 +172,7 @@ redmine_initialize() {
             redmine_conf_set "default.email_delivery.smtp_settings.user_name" "$REDMINE_SMTP_USER"
             redmine_conf_set "default.email_delivery.smtp_settings.password" "$REDMINE_SMTP_PASSWORD"
             # Remove 'USER@' part from e-mail address and use as domain
-            redmine_conf_set "default.email_delivery.smtp_settings.domain" "${REDMINE_SMTP_USER//*@/}"
+            redmine_conf_set "default.email_delivery.smtp_settings.domain" "${REDMINE_SMTP_DOMAIN:-${REDMINE_SMTP_USER//*@/}}"
```
Comment (FraPazGal) [inline bitnami/redmine/5/debian-11/rootfs/opt/bitnami/scripts/libredmine.sh:175]: We should update this comment to reflect the possibility of using `$REDMINE_SMTP_DOMAIN`

File: bitnami/redmine/README.md [hunk 2]
```diff
@@ -274,6 +274,7 @@ To configure Redmine to send email using SMTP you can set the following environm
 - `REDMINE_SMTP_HOST`: SMTP host.
 - `REDMINE_SMTP_PORT`: SMTP port.
 - `REDMINE_SMTP_USER`: SMTP account user.
+- `REDMINE_SMTP_DOMAIN`: SMTP domain.
```
Comment (FraPazGal) [inline bitnami/redmine/README.md:277]: In the same vein, I think we should detail this is to be used when `$REDMINE_SMTP_USER` doesn't apply.

**Review by FraPazGal — COMMENTED** [review]
Hello @dolohow! I have added a couple of minor comments as a review, please check them when you can

**Comment by jqiuyin** [comment]
Is there any progress on this Pull Request?

**Comment by dgomezleon** [comment]
Hi @dolohow ,

Would you like to update the PR to apply for branch 6 and use Fran's suggestions?

**Comment by dgomezleon** [comment]
Due to the lack of activity, we close this PR. We will add the SMTP_DOMAIN env variable separately today.

**Pull request closed without merging**